site/docs/skylark/backward-compatibility.md

```diff
@@ -79,6 +80,15 @@ a dict will fail.
 *   Default: `false`
 *   Tracking issue: [#8730](https://github.com/bazelbuild/bazel/issues/8730)
 
+### Frozen mutable types are no longer hashable
+
+When this flag is enabled, mutable types (e.g. list, dict) do not become
+hashable when frozen and thus cannot be used as dictionary keys.
+
+*   Flag: `--incompatible_disallow_hashing_frozen_mutables`
+*   Default: `false`
+*   Tracking issue: [#7800](https://github.com/bazelbuild/bazel/issues/7800)
+
```

Comment:
fyi, I'm removing this page. Instead we just use GitHub issues to track the changes.